src/main/java/com/solvd/review/kafka/parser/XmlParser.java

```diff
@@ -0,0 +1,23 @@
+package com.solvd.review.kafka.parser;
+
+import com.jcabi.xml.XML;
+import com.jcabi.xml.XMLDocument;
+import lombok.SneakyThrows;
+
+import java.util.Objects;
+
+public class XmlParser {
```

Comment:
the idea of a utility class is that we can't create its instance

Reply:
Fixed, I made this class abstract